test/roundtrip/fold-unreachable.txt

```diff
@@ -26,7 +26,12 @@
   (func (result i32)
     i32.const 0
     return
-    i32.eqz))
+    i32.eqz)
+
+  (func
+    ref.null exn
+    throw_ref
+  ))
```

Comment:
Why this addition to this test?  Was the idea to put some unreachable instruction here after the throw?

Reply:
oh, we probably should indeed. but we were just checking if throw_ref folds correctly.